aboutme.html

```diff
@@ -0,0 +1,127 @@
+            </div>
+            <div class="contact-div">
+                <p class="contact-p"> Please contact us per email for any questions about SAARC summit. </p>
+                <a href="#" class="contact-a">saarc@demo.org</a>
+            </div>
+
+        </section>
+        <section class="second-section">
+            <div class="logo-section-holder">
+                <h2 class="logo-h2">SAARC summit logo</h2>
+                <h2 class="logo-text-2">Saarc Summit Countries Logo</h2>
+                <span class="bar-gold"></span>
+                <p class="logo-p">The logo of SAARC summit was selected by membering countries.</p>
+                <p class="logo-text-p">The logo of SAARC summit was selected by membering countries. This logo was created on the first meeting of the SAARC countries. Changes to this logo are added by the SAARC commitee and are monitored by them.</p>
+                <div class="saarc-img-holder">
+                    <img src="./assets/images/logo.png" alt="SAARC logo">
+                </div>
+            </div>
+            <span class="divider"></span>
+            <div class="summit-info-holder">
+                <h2 class="show-saarc-h2">See the past SAARC summits</h2>
+                <span class="bar-gold"></span>
+                <p class="show-saarc-p">Take a look at two SAARC summits which were in</p>
+                <div class="summits-photo-holder">
+                <div class="first-summit-img">
+                    <img src="./assets/images/about_2011_hover.png" alt="Saarc Summit 2014">
+                </div>
+                <div class="second-summit-img">
+                    <img src="./assets/images/about_2013_hover.png" alt="Saarc Summit 2008">
+                </div>
+                </div>
+            </div>
+        </section>
+        <section class="partner-section">
+            <h2 class="partner-header">Partners</h2>
+            <span class="bar-gold"></span>
+            <div class="partner-section-flag">
+            <div class="partner-1-img">
+                <img src="./assets/images/afghanistan.png" alt="">
+            </div>
+            <div class="partner-1-img">
+                <img src="./assets/images/pakistan.png" alt="">
+            </div>
+            <div class="partner-1-img">
+                <img src="./assets/images/india.png" alt="">
+            </div>
+            <div class="partner-1-img">
+                <img src="./assets/images/bangladesh.png" alt="">
+            </div>
+            </div>
+        </section>
+        <section class="third-section"></section>
+    </main>
+    <footer>
+        <div class="footer-saarc">
+            <div class="footer-img-saarc">
+            <img src="./assets/images/logofooter.png" alt="saarc-summit">
+            </div>
+            <p class="saarc-footer-p">Saarc Summit 2022</p>
+        </div>
+    </footer>
```

Comment:
- [ ] The footer in about page should have a dark background according to requirements. Please fix this 🙏 
- [ ] Please also reduce the size of the footer logo and text to match the given design 🎨 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/176976956-0fb427d6-ba8b-41a0-8e76-63765975a995.png)
### Behance Design
![image](https://user-images.githubusercontent.com/46839548/176976981-03c65cdb-3c3f-4d05-80ea-6c8c821463a1.png)